README.md

```diff
@@ -1,57 +1,11 @@
-# PlanetaryPy Project
+# [PlanetaryPy]((https://planetarypy.org)
```

Comment:
Shouldn't that be [PlanetaryPy Project](https://planetarypy.org)
On the website we also call ourselves "PlanetaryPy Project"?
And one ( less

Reply:
Good catch.